src/common/main/kotlin/tech/thatgravyboat/skyblockapi/helpers/McPlayer.kt

```diff
@@ -17,11 +20,11 @@ object McPlayer {
 
     val position: Vec3? get() = self?.position()
 
-    val name: String get() = McClient.self.gameProfile.name
-    val uuid: UUID get() = McClient.self.gameProfile.id
-    val skin: PlayerSkin? get() = McClient.self.player?.skin
+    val name: String get() = McClient.self.gameProfile.name()
+    val uuid: UUID get() = McClient.self.gameProfile.id()
+    val skin: PlayerSkin? = McClient.self.player?.skin()
```

Comment:
```suggestion
    val skin: PlayerSkin? get() = McClient.self.player?.skin()
```
pleading why remove it

Reply:
you can change your skin without needing to reboot (boot, change it in the launcher, rejoin the server, and its changed)